run-spartacus-cpq-e2es.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# install cypress standlone dependencies
+(cd projects/storefrontapp-e2e-cypress && yarn install)
```

Comment:
I guess this can be improved where we pass in args to determine what package script to run.

You don't have to fix it and add that switch statement or conditional statements for the args, but I'm just mentioning that this won't scale well as more teams test in ccv2.

I believe the team will 'upgrade' this pattern when we test 5.0 in ccv2

Reply:
I agree with this statement. Should I change it or do we leave it for now?

Reply:
We can leave it for now, and someone in Spartacus team will update.

However, if you feel like updating it yourself, please feel free to do so! Help is always much appreciated!